src/migrations/shared.lua

```diff
@@ -16,9 +17,11 @@ shared.STACKABLES = {
     { item = "ground-sample01", icon = nil, stage = 1 },
     { item = "lens", icon = nil, stage = 1 },
     { item = "raw-borax", icon = nil, stage = 1 },
-    { item = "nexelite-ore", icon = nil, stage = 2 },
+    { item = "sand", icon = nil, stage = 1 },
+    { item = "pure-sand", icon = nil, stage = 1 },
+    { item = "nexelit-ore", icon = nil, stage = 2 },
     { item = "niobium-ore", icon = nil, stage = 2 },
-    { item = "limestone", icon = nil, stage = 2 }
+    { item = "limestone", icon = nil, stage = 1 }
```

Comment:
Why move it to stage 1? Is it needed before you can build red belts?

Reply:
yep now with py raw ores it is

Reply:
or i could check it if you want

Reply:
Ok, haven't had the time to try it out yet.

Reply:
it is unlocked at the beginning and needed for coal processing 1 witch is stage 1

Reply:
Its Ok. As i've said I've had no time to try it out yet. Since it is needed earlier now its perfectly legit to reduce the stage to 1.